packages/artifact/src/internal/requestUtils.ts

```diff
@@ -0,0 +1,74 @@
+import {IHttpClientResponse} from '@actions/http-client/interfaces'
+import {isRetryableStatusCode, isSuccessStatusCode, sleep} from './utils'
+import * as core from '@actions/core'
+
+export async function retry<T>(
+  name: string,
+  method: () => Promise<T>,
+  getStatusCode: (response: T) => number | undefined,
```

Comment:
When would a response not have a status code?

Reply:
Maybe the call was never made?

Reply:
It is possible that the response message can be undefined, and thus there is no status code. Changing this to just `number` produces the following error

![image](https://user-images.githubusercontent.com/16109154/102517657-99848800-4090-11eb-8637-1ef5f86d039e.png)